helpers/bills.js

```diff
@@ -248,16 +249,20 @@ const getDraftBillsPerSubscription = (events, customer, subscription, fundings,
     if (moment(event.startDate).isBefore(startDate)) startDate = moment(event.startDate);
   }
 
+  const serviceMatchingVersion = getMatchingVersion(query.startDate, subscription.service);
   const draftBillInfo = {
+    _id: mongoose.Types.ObjectId(),
     subscription,
     identity: customer.identity,
     discount: 0,
     startDate: startDate.toDate(),
     endDate: moment(query.endDate, 'YYYYMMDD').toDate(),
     unitExclTaxes: getExclTaxes(
       getMatchingVersion(query.startDate, subscription).unitTTCRate,
-      getMatchingVersion(query.startDate, subscription.service).vat
+      serviceMatchingVersion.vat
     ),
+    vat: serviceMatchingVersion.vat,
+    editDiscount: false,
```

Comment:
je n'ai pas trop ce champs en back car a mon sens ça n'a qu'une utilité cote front et notamment je ne sais pas si ca respecte le principe de `Separation of concerns`

Reply:
Done !